utilities/backupable/backupable_db.cc

```diff
@@ -986,7 +996,7 @@ Status BackupEngineImpl::CreateNewBackupWithMetadata(
 
   std::vector<BackupAfterCopyOrCreateWorkItem> backup_items_to_finish;
   // Add a CopyOrCreateWorkItem to the channel for each live file
-  db->DisableFileDeletions();
+  db->DisableFileDeletions().PermitUncheckedError();
```

Comment:
I don't know the implications of Enable/DisableFileDeletions failing. If it is okay to ignore them, can you add a comment?

Reply:
They never currently fail.  Restructured the code slightly